libs/payments/cart/src/lib/cart.service.ts

```diff
@@ -639,6 +643,31 @@ export class CartService {
             price,
             cartDetails.currency || oldCart.currency
           );
+
+          if (oldCart.stripeCustomerId) {
+            try {
+              const customer = await this.customerManager.retrieve(
+                oldCart.stripeCustomerId
+              );
+              await this.invoiceManager.previewUpcoming({
+                priceId: price.id,
+                currency: cartDetails.currency || oldCart.currency,
+                customer,
+                taxAddress: cartDetails.taxAddress,
```

Comment:
question: Why use `cartDetails.taxAddress` instead of the value on the cart?

Reply:
I don't think we actually need taxAddress at all.